hw/ip/flash_ctrl/flash_ctrl_pkg.core

```diff
@@ -23,7 +23,35 @@ filesets:
       - rtl/flash_phy_pkg.sv
     file_type: systemVerilogSource
 
+  files_verilator_waiver:
```

Comment:
Hey Tim, i don't know this is correct approach or not. Is it ok to include lint waiver in flash_pkg ?

I should figure out why KMAC depends on the flash_pkg.core but this is basically lint does not include flash.waiver so that it errors out on `flash_ctrl_pkg.sv` file.

Reply:
Ah, this isn't quite right, I don't think. Files like `lint/flash_ctrl.waiver` already belong to `flash_ctrl.core`. I think the correct approach is to add a `lint/flash_ctrl_pkg.waiver` containing the last waiver that's currently in `lint/flash_ctrl.waiver` and then update this core to include that.

Reply:
Make sense. Revised now.

Reply:
let me know if you need my help to dig around why there's a flash dependency here.
could it be you're depending on keymgr somewhere that's also depending on flash?